storage/src/main/java/org/apache/kafka/storage/internals/log/LogFileUtils.java

```diff
@@ -32,4 +37,31 @@ public static long offsetFromFileName(String fileName) {
         return Long.parseLong(fileName.substring(0, fileName.indexOf('.')));
     }
 
+    /**
+     * Returns a File instance with parent directory as logDir and the file name as producer snapshot file for the
+     * given offset.
+     *
+     * @param logDir parent directory for the snapshot file
+     * @param offset offset number for the snapshot file
+     * @return a File instance for producer snapshot.
```

Comment:
This documentation is different from the `UnifiedLog` one. In particular, the details for `offset` are important (and now missing).